src/script.js

```diff
@@ -86,7 +86,7 @@ $j(document).ready(() => {
 export function getGameConfig() {
 	let defaultConfig = {
 		playerMode: $j('input[name="playerMode"]:checked').val() - 0,
-		creaLimitNbr: $j('input[nlogame="activeUnits"]:checked').val() - 0, // DP counts as One
+		creaLimitNbr: $j('input[ngame="activeUnits"]:checked').val() - 0, // DP counts as One
```

Comment:
Is there actually an `ngame` attribute here? I suspect it was supposed to be `name` ;)